instrumentation/dalli/lib/opentelemetry/instrumentation/dalli/patches/server.rb

```diff
@@ -14,9 +14,9 @@ def request(op, *args)
             operation = Utils.opname(op, multi?)
             attributes = {
               'db.system' => 'memcached',
-              'db.statement' => Utils.format_command(operation, args),
               'net.peer.name' => hostname,
-              'net.peer.port' => port
+              'net.peer.port' => port,
+              **db_statement(operation, args),
             }
             attributes['peer.service'] = config[:peer_service] if config[:peer_service]
```

Comment:
The double splat there doesn't idiomatic, I totally believe it works but it's awkward looking :)
```suggestion
            }
            attributes['db.statement'] = db_statement if config[:db_statement] == :include || config[:db_statement] == :obfuscate
            attributes['peer.service'] = config[:peer_service] if config[:peer_service]
```